iOS/ExposureNotification/source/ApiDefinition.cs

```diff
@@ -106,6 +114,15 @@ interface ENExposureConfiguration {
 		[Export ("minimumRiskScore")]
 		byte MinimumRiskScore { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 6)]
+		[Export ("minimumRiskScoreFullRange")]
+		double MinimumRiskScoreFullRange { get; set; }
+
+		[Introduced (PlatformName.iOS, 13, 6)]
+		[BindAs (typeof (int []))]
```

Comment:
What does the `[BindAs (typeof (int []))]` actually do when the return type is `NSNumber[]`?

Reply:
This means that we will take care of the conversion of `NSNumber []` <-> `int []` automatically in the binding code so the signature will become

```csharp
int [] AttenuationDurationThresholds { get; }
```

So you do not need to worry about the NSNumber [] detail

Reply:
That is very nice.